src/conn/conn_tiered.c

```diff
@@ -307,21 +351,39 @@ __wt_flush_tier(WT_SESSION_IMPL *session, const char *config)
 {
     WT_CONFIG_ITEM cval;
     WT_DECL_RET;
+    uint32_t flags;
     const char *cfg[3];
-    bool force;
 
     WT_STAT_CONN_INCR(session, flush_tier);
     if (FLD_ISSET(S2C(session)->server_flags, WT_CONN_SERVER_TIERED_MGR))
         WT_RET_MSG(
           session, EINVAL, "Cannot call flush_tier when storage manager thread is configured");
 
+    flags = 0;
     cfg[0] = WT_CONFIG_BASE(session, WT_SESSION_flush_tier);
     cfg[1] = (char *)config;
     cfg[2] = NULL;
     WT_RET(__wt_config_gets(session, cfg, "force", &cval));
-    force = cval.val != 0;
+    if (cval.val)
+        flags |= WT_FLUSH_TIER_FORCE;
```

Comment:
Use the `LF_SET` macros for this and the following similar statements.

Reply:
Thanks! Fixed.